### PR TITLE
doc: Release notes for 1.17.0, 1.16.2, and 1.15.4

### LINKED ISSUE #687: Release notes for 1.17.0, 1.16.2, and 1.15.4
Establish EOL for 1.15 (Oct 6)

### PR DESCRIPTION
**Pull request by gbartolini**
Closes #687

Signed-off-by: Gabriele Bartolini <gabriele.bartolini@enterprisedb.com>

### CODE REVIEW

**Review by theadamwright — APPROVED** [review]

**Review by theadamwright — APPROVED** [review]

File: docs/src/supported_releases.md [hunk 1]
```diff
@@ -77,12 +78,12 @@ Kubernetes distribution, like Red Hat OpenShift (this might change in the
...
 
-## Upcoming release
+## Upcoming releases
 
 | Version         | Release Date       | End of Life              | Supported Kubernetes Versions |
 |-----------------|--------------------|--------------------------|-------------------------------|
-| 1.17.0          | September 15, 2022 | ~ February 10, 2023      | 1.22, 1.23, 1.24              |
-| 1.18.0          | November 10, 2022  | 1 month after 1.20       |                               |
+| 1.18.0          | October 25, 2022   | 1 month after 1.20       | 1.23, 1.24, 1.25              |
+| 1.19.0          | October 25, 2022   | 1 month after 1.21       |                               |
```
Comment (gbartolini) [inline docs/src/supported_releases.md:86]:
```suggestion
| 1.19.0          | January 12, 2023   | 1 month after 1.21       |                               |
```

File: docs/src/supported_releases.md [hunk 2]
```diff
@@ -77,12 +78,12 @@ Kubernetes distribution, like Red Hat OpenShift (this might change in the
...
 
-## Upcoming release
+## Upcoming releases
 
 | Version         | Release Date       | End of Life              | Supported Kubernetes Versions |
 |-----------------|--------------------|--------------------------|-------------------------------|
-| 1.17.0          | September 15, 2022 | ~ February 10, 2023      | 1.22, 1.23, 1.24              |
-| 1.18.0          | November 10, 2022  | 1 month after 1.20       |                               |
+| 1.18.0          | October 25, 2022   | 1 month after 1.20       | 1.23, 1.24, 1.25              |
+| 1.19.0          | January 12, 2023   | 1 month after 1.21       |                               |
```
Comment (gbartolini) [inline docs/src/supported_releases.md:86]:
```suggestion
| 1.18.0          | October 25, 2022   | April 13, 2023       | 1.23, 1.24, 1.25              |
| 1.19.0          | January 12, 2023   | TBD       |                               |
| 1.20.0          | March 13, 2023   | TBD       |                               |
```

File: docs/src/supported_releases.md [hunk 3]
```diff
@@ -65,9 +65,10 @@ Git tags for versions are prepended with `v`.
 
 | Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported |
 |-----------------|----------------------|-------------------|--------------------------|-------------------------------|---------------------------|
-| main            | No, development only |                   |                          |                               |                           |
-| 1.15.x          | Yes                  | April 21, 2022    | October 15, 2022         | 1.21, 1.22, 1.23              | 1.19, 1.20, 1.24          |
+| 1.17.x          | Yes                  | September 6, 2022 | ~ February 10, 2023      | 1.22, 1.23, 1.24              | 1.19, 1.20, 1.21          |
 | 1.16.x          | Yes                  | July 7, 2022      | ~ December 10, 2022      | 1.22, 1.23, 1.24              | 1.19, 1.20, 1.21          |
```
Comment (gbartolini) [inline docs/src/supported_releases.md:69]:
```suggestion
| 1.17.x          | Yes                  | September 6, 2022 | ~ February 12, 2023      | 1.22, 1.23, 1.24              | 1.19, 1.20, 1.21          |
| 1.16.x          | Yes                  | July 7, 2022      | ~ November 25, 2022      | 1.22, 1.23, 1.24              | 1.19, 1.20, 1.21          |
```

**Review by mnencia — APPROVED** [review]

**Review by armru — APPROVED** [review]

**Review by jsilvela — APPROVED** [review]

**Review by leonardoce — APPROVED** [review]

File: docs/src/release_notes/v1.15.md [hunk 4]
```diff
@@ -6,6 +6,30 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.15)
 on the release branch in GitHub.
 
+!!! Warning
+    End-of-Life (EOL) date for version 1.15 is 6 October, 2022.
```
Comment (sxd) [inline docs/src/release_notes/v1.15.md:10]: shouldn't this be October 6th or 6 of October?

**Review by sxd — APPROVED** [review]

**Pull request merged**